### PR TITLE
Landing page update

### PR DESCRIPTION
**Pull request by carlosgauci**
WIP Landing page update

### CODE REVIEW

**Comment by TuanaCelik** [comment]
@carlosgauci - on top of the edit requests from the designers, we could also resolve the issues mentioned by @bilgeyucel here in the same PR? https://github.com/deepset-ai/haystack-home/issues/303

**Comment by TuanaCelik** [comment]
@carlosgauci as mentioned on Discord too, I've added a 'release notes' section to the PR too. I'm using the 'blog' layout but for some reason I can't get the page here to display the cards properly. This page would probably be re-worked in future but we want something simple for next week. https://haystack-home-git-landing-page-update-deepset-overnice.vercel.app/release-notes

The image isn't seeming to be displayed properly :/

**Comment by bilgeyucel** [comment]
Great updates! 🚀 

@carlosgauci One minor comment: sidebar tags are still visible for intermediate tutorials 
image

**Comment by TuanaCelik** [comment]
I can fix that @carlosgauci @bilgeyucel dw. Making a commit now

**Comment by bilgeyucel** [comment]
@TuanaCelik tag is still visible when I'm on that tutorial's page
image

**Comment by TuanaCelik** [comment]
should be fixed now @bilgeyucel

File: content/overview/quick-start.md [hunk 1]
```diff
@@ -3,208 +3,235 @@ layout: overview
...
-[{'answer': 'The father of Arya Stark is Lord Eddard Stark of '
-                'Winterfell. [Document 1, Document 4, Document 5]'}]
-```
+{{< /tabs >}}
 
-For a hands-on guide on how to build your first RAG Pipeline, see our tutorial.
+For a hands-on guide on how to build your first RAG Pipeline with Haystack 2.0, see our tutorial.
 
-{{< button url="https://haystack.deepset.ai/tutorials/22_pipeline_with_promptnode" text="Tutorial: Creating a RAG Pipeline" color="green">}}
+{{< button url="https://haystack.deepset.ai/tutorials/27_first_rag_pipeline" text="Tutorial: Creating a RAG Pipeline" color="green">}}
```
Comment (bilgeyucel) [inline content/overview/quick-start.md:237]: Do we plan to add traceable links to these buttons @TuanaCelik? the other button is for "Docs: Installation"

Comment (TuanaCelik) [reply content/overview/quick-start.md:237]: O M G good catch!!!  thank you! And we can, but it's all internal links at this point so not sure how useful they will be. I will add it anyway.

**Pull request merged**